AYIM & MCA 2018 newspost (#2623)
ppy/osu-wiki: news/2019-09-29-ayim-mca-2018-newspost.md

---
layout: post
title: A Year in Mapping 2018 and Mapper's Choice Awards 2018
date: 2019-09-29 12:00:00 +0000
---

While a bit late, the annual Mapper's Choice Awards and A Year in Mapping summary for 2018 are now here! 

![](IMAGE COMING SOON)

> # Mapper's Choice Awards

Reviewer: ```suggestion
## Mapper's Choice Awards
```

Mapper's Choice Awards is an event where eligible users can participate in nominating and then voting on the best maps of each of the four game modes as well as storyboards. This event is meant to give perspective as to what the best maps are of 2018 in the eyes of the mapping community! For this event, storyboarding will be regarded as a mode to give the category more focus. Further information about this event can be found on the forum thread.

Each mode will have two **Grand MCA Awards**, one for the best beatmap, and one for the best user. In addition, each mode will have other categories where maps can compete for the top spot such as marathon, spread, and and different music genres. Depending on the mode, there will be other additional categories for voting as well.

Even if you are not able to vote, feel free to follow along with the events! The results will be later on released via livestreams, in which commentators will discuss the results.

> ## Schedule

Reviewer: ```suggestion
### Schedule
```

As of right now, Mapper's Choice Awards is in the nomination stage, and the rest of the schedule can be seen below. If you're eligible to vote or nominate, be sure to do so while you still can!

- The **Nomination Stage** is held from **September 28th, 00:00 UTC to October 5th, 00:00 UTC** (7 DAYS)
- The **Voting Stage** is held from **October 7th, 00:00 UTC to October 17th, 18:00 UTC** (11 DAYS)
- The **Taiko / Catch / Mania Stream** is held on **October 19th**, TIME TBD
- The **Storyboard / Standard Stream** is held on **October 20th**, TIME TBD

# A Year In Mapping 2018

[A Year In Mapping 2018 forum thread](https://osu.ppy.sh/community/forums/topics/966004)

> A Year In Mapping is back again to bring you the achievements and statistics of the mapping community this past year. Each mode is given a section of statistics from information like who ranked the most maps in a mode to the most popular artists and songs. Like 2017, 2018's edition of A Year In Mapping has written commentary on users from each mode who are eligible for comments. If you want to learn more about different members of the mapping community, be sure to check out the thread and given it a read!

Reviewer: ```suggestion
A Year In Mapping (*AYIM*) is back again to bring you the achievements and statistics of the mapping community in 2018. Each mode is given a section of statistics, such as who ranked the most maps and most popular artists and songs. Like 2017, 2018's edition of A Year In Mapping features commentary about mapping community members from each mode. Check out the thread if you're interested!
```
